deploy/crd.yaml

```diff
@@ -412,26 +412,30 @@ spec:
       type: string
       description: The current state machine phase for this FlinkApplication
       JSONPath: .status.phase
+    - name: Application Version
+      type: string
+      description: The version of the Flink cluster
+      JSONPath: .status.appStatus[*].version
     - name: Cluster Health
       type: string
       description: The health of the Flink cluster
-      JSONPath: .status.clusterStatus.health
+      JSONPath: .status.appStatus[*].clusterStatus.health
```

Comment:
What I have here doesn't quite work for multi app statuses. Printer columns only seem to support `boolean,date,integer,number,string` and also don't support [JSONPath operators/expressions](https://kubernetes.io/docs/reference/kubectl/jsonpath/) :( 
I think to surface multi app statuses, I'd have to explicitly specify the array index and the output would look something like:
```
NAME                PHASE         APPLICATION VERSION   CLUSTER HEALTH   JOB HEALTH   JOB RESTARTS   APPLICATION VERSION   CLUSTER HEALTH   JOB HEALTH   JOB RESTARTS   AGE
operator-test-app   DualRunning   green                 Green            Green                       blue                  Green            Green                       6h4
```

This would mean a few empty columns when there's only 1 application running.